google/cloud/accesscontextmanager/README.md

```diff
@@ -0,0 +1,91 @@
+# Access Context Manager API C++ Client Library
+
+:construction:
+
+This directory contains an idiomatic C++ client library for the
+[Access Context Manager API][cloud-service-docs], a service for setting
+attribute based access control on requests to GCP services.
+
+This library is **experimental**. Its APIS are subject to change without notice.
+
+Please note that the Google Cloud C++ client libraries do **not** follow
+[Semantic Versioning](https://semver.org/).
+
+## Supported Platforms
+
+* Windows, macOS, Linux
+* C++11 (and higher) compilers (we test with GCC >= 5.4, Clang >= 6.0, and
+  MSVC >= 2017)
+* Environments with or without exceptions
+* Bazel and CMake builds
+
+## Documentation
+
+* Official documentation about the [Access Context Manager API][cloud-service-docs] service
+* [Reference doxygen documentation][doxygen-link] for each release of this
+  client library
+* Detailed header comments in our [public `.h`][source-link] files
+
+[cloud-service-docs]: https://cloud.google.com/accesscontextmanager
```

Comment:
404. You want: https://cloud.google.com/access-context-manager/docs

Reply:
Fixed here and below.